packages/db/postgres/tables.sql

```diff
@@ -38,9 +47,10 @@ CREATE TABLE IF NOT EXISTS public.user_tag
   id              BIGSERIAL PRIMARY KEY,
   user_id         BIGINT                                                        NOT NULL REFERENCES public.user (id),
   tag             user_tag_type                                                 NOT NULL,
-  -- tag_value is useful for certain tags like STORAGE_LIMIT e.g. tag="STORAGE_LIMIT", tag_value="1TB"
-  tag_value       TEXT                                                                  ,
+  value           TEXT                                                          NOT NULL,
+  value_type      user_tag_value_type                                           NOT NULL,
   inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now())     NOT NULL,
+  updated_at  TIMESTAMP WITH TIME ZONE                                                  ,
```

Comment:
Let's talk this sync, but for arguments in the other PR, probably you were right in not having it and relying on delete + create again.